modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1312,12 +1267,35 @@ The implementation is based on @cite MOSSE Visual Object Tracking using Adaptive
 
 class CV_EXPORTS_W TrackerMOSSE : public Tracker
 {
- public:
-  /** @brief Constructor
+public:
+    struct CV_EXPORTS Params {
+        /**
+    * \brief Constructor
+    */
+        Params();
+
+        /**
+    * \brief Read parameters from a file
+    */
+        void read(const FileNode& /*fn*/);
+
+        /**
+    * \brief Write parameters to a file
+    */
+        void write(FileStorage& /*fs*/) const;
+
+
+        double eps;            // for normalization
+        double rate;           // learning rate
+        double psr_threshold;  // no detection, if PSR is smaller than this
+    };
+    /** @brief Constructor
   */
-  CV_WRAP static Ptr<TrackerMOSSE> create();
+    CV_WRAP static Ptr<TrackerMOSSE> create();
+
+    static Ptr<TrackerMOSSE> create(const TrackerMOSSE::Params& params);
```

Comment:
`Params` are not accessible from Python, so probably we need one more `CV_WRAP` overload with "unpacked" parameters (which creates `Params` internally).